src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Project/Project+CBXMLHandler.m

```diff
@@ -44,7 +44,13 @@ + (instancetype)parseFromElement:(GDataXMLElement*)xmlElement withContext:(CBXML
     // IMPORTANT: DO NOT CHANGE ORDER HERE!!
     project.header = [self parseAndCreateHeaderFromElement:xmlElement withContext:context];
     project.userData = [self parseAndCreateVariablesFromElement:xmlElement withContext:context];
-    project.objectList = [self parseAndCreateObjectsFromElement:xmlElement withContext:context];
+    for (SpriteObject *object in [self parseAndCreateObjectsFromElement:xmlElement withContext:context]) {
```

Comment:
Please remove the whole parsing of objects to` Scene+CBXMLHandler`(i.e., `parseAndCreateObjectsFromElement`).